### PR TITLE
Fix stripejs input names

### LINKED ISSUE #118: Remove name attrs on fields for StripeCardProvider
The name attrs present on the fields mean those inputs are submitted to the server, which means the server owner needs to have PCI compliance, the exact problem the stripe.js library is trying to avoid.

In `payments/static/js/payments/stripe.js`, the following lines:

```
        name: this.elements['name'].value,
        number: this.elements['number'].value,
        cvc: this.elements['cvv2'].value,
        exp_month: this.elements['expiration_0'].value,
        exp_year: this.elements['expiration_1'].value,
```

Simply need changing to

```
        name: this.elements.id_name.value,
        number: this.elements.id_number.value,
        cvc: this.elements.id_cvv2.value,
        exp_month: this.elements.id_expiration_0.value,
        exp_year: this.elements.id_expiration_1.value,
```

And the input widgets for those elements in django need their render method overriding to remove the `name=...` from the HTML.

### PR DESCRIPTION
**Pull request by artursmet**
Closes #118

### CODE REVIEW

**Comment by patrys** [comment]
Why is this needed?

**Comment by artursmet** [comment]
@patrys As it's mentioned in #118, we have to pass sensitive data only to stripe, and skip django application during the process.

**Comment by artursmet** [comment]
@patrys I think I finally managed to implement it properly. As a bonus we have `SensitiveTextInput` and `SensitiveSelect` widgets, that are rendered without `name` attribute, so they can be reused with another payment gateways, that use javascript processing.

File: payments/fields.py [hunk 1]
```diff
@@ -66,6 +66,9 @@ class CreditCardExpiryField(forms.MultiValueField):
     EXP_YEAR = [(str(x), str(x)) for x in range(date.today().year,
                                                 date.today().year + 15)]
 
+    EXP_MONTH_CHOICES = [('', _('Month'))] + EXP_MONTH
+    EXP_YEAR_CHOICES = [('', _('Year'))] + EXP_YEAR
```
Comment (patrys) [inline payments/fields.py]: Fun fact: these are calculated only while the application starts.

**Comment by patrys** [comment]
Can we move the `classmethod`s to module-level functions? `classmethod` is mostly intended to provide alternative class constructors/initializers.

**Pull request merged**